Client/src/main/java/org/projectcardboard/client/controller/GameController.java

```diff
@@ -140,7 +144,8 @@ public void move(Troop selectedTroop, int row, int column) {
           Message.makeMoveTroopMessage(SERVER_NAME, selectedTroop.getCard().getCardId(), target);
       Client.getInstance().addToSendingMessagesAndSend(message);
     } catch (InputException e) {
-      System.out.println(e.getMessage());
+      logger.warn("Failed to move troop");
```

Comment:
Similar thing here, there's info that could be useful at the warn level about the troop that could give a hint about why it failed without the stack trace.